db/migrate/20171012024407_create_votes.rb

```diff
@@ -0,0 +1,23 @@
+class CreateVotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :votes do |t|
+      t.integer :vote
+      t.string :votable_type
+      t.integer :votable_id
+      t.integer :user_id, { null: false }
```

Comment:
Let's get a `null: false` on `votable_id` and `votable_type` also, since we would not want to save a vote record that is not associated with a comment or suggestion